game-core/src/test/java/games/strategy/triplea/delegate/RevisedTest.java

```diff
@@ -104,7 +104,6 @@ private static void givenRemotePlayerWillSelectCasualtiesPer(
         any(),
         any(),
         any(),
-        any(),
         anyBoolean(),
         any(),
         any(),
```

Comment:
This is where the test failure comes from.
In line 90, the 11th argument is being cast to a `CasualtyList`, after this change the parameter is the 10th argument (when starting from 0, as usual).